modules/components/AsyncConnect.js

```diff
@@ -44,7 +44,9 @@ export default class AsyncConnect extends Component {
   }
 
   componentWillReceiveProps(nextProps) {
-    this.loadAsyncData(nextProps);
+    if (!this.context.store.getState().reduxAsyncConnect.loading) {
```

Comment:
`componentWillReceiveProps` is not called on the initial render

https://facebook.github.io/react/docs/component-specs.html#updating-componentwillreceiveprops